pkg/entities/alerts.go

```diff
@@ -336,21 +336,21 @@ func (a *StateHashAlert) Level() string {
 }
 
 type AlertFixed struct {
-	Timestamp int64 `json:"timestamp"`
-	Fixed     Alert `json:"fixed"`
+	Timestamp int64  `json:"timestamp"`
+	Fixed     string `json:"fixed"`
 }
 
 func (a *AlertFixed) ShortDescription() string {
 	return AlertFixedNotification
 }
 
 func (a *AlertFixed) ID() string {
-	digest := crypto.MustFastHash([]byte(a.ShortDescription() + a.Fixed.ID()))
+	digest := crypto.MustFastHash([]byte(a.ShortDescription() + a.Fixed))
```

Comment:
You must not use result of `Message()` call in `AlertFixed.ID` method. Only `ID` or any **static** data is allowed.

An alert can have different message strings but only one ID.